pkg/clusterversion/cockroach_versions.go

```diff
@@ -188,8 +188,8 @@ const (
 	// V25_1 is CockroachDB v25.1. It's used for all v25.1.x patch releases.
 	V25_1
 
-	// V25_2 is CockroachDB v25.2. It's used for all v25.2.x patch releases.
-	V25_2
+	// TODO_Delete_V25_2 is CockroachDB v25.2. It's used for all v25.2.x patch releases.
+	TODO_Delete_V25_2
```

Comment:
nit: do we really want to remove `V25_2`? Looks like we did keep around final older versions in the codebase.